requirements.txt

```diff
@@ -0,0 +1,6 @@
+requests
+python-dotenv
+python-telegram-bot
+ansistrip
+pygithub
+proxyscrape
```

Comment:
Maybe you could also write the version numbers (just in case there's a breaking change in any of these packages)
[Here's a Sample](https://github.com/akigugale/covid19india-telegram-bot/blob/master/requirements.txt)